src/libs/runtime-utils.js

```diff
@@ -347,9 +355,60 @@ export const isCurrentGalaxyDiskDetaching = apps => {
 export const getGalaxyCostTextChildren = (app, appDataDisks) => {
   const dataDisk = getCurrentAttachedDataDisk(app, appDataDisks)
   return app ?
-    [getComputeStatusForDisplay(app.status), dataDisk ? ` (${Utils.formatUSD(getGalaxyCost(app, dataDisk))} / hr)` : ``] : ['None']
+    [getComputeStatusForDisplay(app.status), dataDisk ? ` ${Utils.formatUSD(getGalaxyCost(app, dataDisk))}/hr` : ``] : ['']
+}
+
+// TODO: multiple runtime: this is a good example of how the code should look when multiple runtimes are allowed, over a tool-centric approach
+export const getCostDisplayForTool = (app, appDataDisks, currentRuntime, currentRuntimeTool, toolLabel) => Utils.cond(
+  [toolLabel === tools.Galaxy.label, () => getGalaxyCostTextChildren(app, appDataDisks)],
+  [toolLabel === tools.Cromwell.label, () => ''], // We will determine what to put here later
+  [toolLabel === tools.Azure.labels, () => ''], //TODO: Azure cost calculation
+  [getRuntimeForTool(toolLabel, currentRuntime, currentRuntimeTool), () => {
+    const runtime = getRuntimeForTool(toolLabel, currentRuntime, currentRuntimeTool)
+    const totalCost = getRuntimeCost(runtime)
+    return `${getComputeStatusForDisplay(runtime.status)} ${Utils.formatUSD(totalCost)}/hr`
+  }],
+  [Utils.DEFAULT, () => {
+    return ''
+  }]
+)
+
+export const getCostForTool = (app, appDataDisks, currentRuntime, currentRuntimeTool, toolLabel) => Utils.cond(
+  [toolLabel === tools.Galaxy.label, () => getGalaxyCost(app, appDataDisks)],
+  [toolLabel === tools.Cromwell.label, () => ''], // We will determine what to put here later
+  [toolLabel === tools.Azure.labels, () => ''], //TODO: Azure cost calculation
+  [getRuntimeForTool(toolLabel, currentRuntime, currentRuntimeTool), () => {
+    const runtime = getRuntimeForTool(toolLabel, currentRuntime, currentRuntimeTool)
+    return getRuntimeCost(runtime)
+  }],
+  [Utils.DEFAULT, () => {
+    return ''
+  }]
+)
+
+export const getCostDisplayForDisk = (app, appDataDisks, computeRegion, currentRuntimeTool, persistentDisks, runtimes, toolLabel) => {
+  const diskCost = getCostForDisk(app, appDataDisks, computeRegion, currentRuntimeTool, persistentDisks, runtimes, toolLabel)
+  return diskCost ? `Disk ${Utils.formatUSD(getCostForDisk(app, appDataDisks, computeRegion, currentRuntimeTool, persistentDisks, runtimes, toolLabel))}/hr` : ''
```

Comment:
Why do you call `getCostForDisk` again instead of using the variable you just defined?